pyvrp/cpp/ProblemData.h

```diff
@@ -82,22 +98,29 @@ class ProblemData
      * @param servDurs     Service durations.
      * @param distMat      Distance matrix.
      */
-    ProblemData(std::vector<std::pair<int, int>> const &coords,
-                std::vector<int> const &demands,
-                size_t numVehicles,
-                size_t vehicleCap,
-                std::vector<std::pair<int, int>> const &timeWindows,
-                std::vector<int> const &servDurs,
-                std::vector<std::vector<int>> const &distMat);
+    ProblemData(
+        std::vector<std::pair<int, int>> const &coords,
+        std::vector<int> const &demands,
+        size_t numVehicles,
+        size_t vehicleCap,
+        std::vector<std::pair<duration_type, duration_type>> const &timeWindows,
+        std::vector<duration_type> const &servDurs,
+        std::vector<std::vector<duration_type>> const &distMat);
```

Comment:
Shouldn't `distMat` have `distance_type`? (They're the same now anyway)

Reply:
Yes! Funnily enough this was already OK in the bindings, but not yet updated here.